### DISCUSSION TITLE
Fulfill order input_data

### DISCUSSION THREAD

**Discussion by reedchan7** [ProjectOpenSea/opensea-js#1238] [General]
I generated the corresponding Go code based on the `.abi` file of seaport v1.5. Then I called the Fulfill method according to the [OpenSea API](https://docs.opensea.io/reference/api-overview). According to the returned `fulfillment_data.transaction.function`, I called the corresponding ABI method.

For example, for `fulfillBasicOrder_efficient_6GL6yc`, I need to convert `fulfillment_data.transaction.input_data` into the arguments required by the method in the corresponding ABI of `fulfillBasicOrder_efficient_6GL6yc`.

**My problem is that** I see many similar `fulfill` methods in ABI, and each has a different parameter list. However, OpenSea's official documentation does not provide a detailed introduction to this input_data structure, and there is no input_data corresponding to every contract method, or the data structure and extraction method.
How should I handle my requirements? Where can I find more detailed documentation, code, or information?

<img width="710" alt="image" src="https://github.com/ProjectOpenSea/seaport/assets/49066821/4ccb27e6-cb5a-4072-9e04-3b1c109a7301">

<img width="1325" alt="image" src="https://github.com/ProjectOpenSea/seaport/assets/49066821/fe7e9198-83a7-4ac0-8d81-285defdd96cb">

<img width="1726" alt="image" src="https://github.com/ProjectOpenSea/seaport/assets/49066821/b13b6323-0426-4654-b926-285a018bba2b">

Thanks.

**Comment by ashwwwin** [comment]
Did you end up figuring this out?
